src/status_im/common/controlled_input/utils.cljs

```diff
@@ -0,0 +1,98 @@
+(ns status-im.common.controlled-input.utils
+  (:require
+    [clojure.string :as string]
+    [reagent.core :as reagent]))
+
+(defn create-input-state
+  []
+  (reagent/atom {:value       ""
```

Comment:
Should we swap to react state while we refactor?

Reply:
I supposed that keeping `reagent/atom` is what will guarantee that components will be re-rendered when it is changed. Do you know if they will be rendered with clojure atom?

Reply:
react state will re render too. I don't mean an atom btw.

I mean `[value set-value] (rn/use-state {:value""})`

Reply:
Oh, great idea! It should also make code more idiomatic! Thank you)